src/api/grainConfig.js

```diff
@@ -67,6 +67,7 @@ export function toDistributionPolicy(x: GrainConfig): DistributionPolicy {
     allocationPoliciesDeprecated.push({
       budget: toNonnegativeGrain(immediatePerWeek),
       policyType: "IMMEDIATE",
+      numPeriodsLookback: 1, // TODO(eli): no customization until after #2600.
```

Comment:
this should probably be renamed `numIntervalsLookback ` for consistency.

Reply:
Agreed, done.